dll/win32/shlwapi/propbag.cpp

```diff
@@ -1412,11 +1431,50 @@ HKEY CViewStatePropertyBag::_GetHKey(DWORD dwVspbFlags)
     return SHGetShellKey(SHKEY_Key_ShellNoRoam | SHKEY_Root_HKCU, NULL, TRUE);
 }
 
+UINT CViewStatePropertyBag::_GetMRUSize(HKEY hKey)
+{
+    DWORD dwValue, cbValue = sizeof(dwValue);
+
+    if (SHGetValueW(hKey, NULL, L"BagMRU Size", NULL, &dwValue, &cbValue) != ERROR_SUCCESS)
+        return 400;
```

Comment:
What would this value correspond to?

Reply:
This is the default size of MRU (most recently used) list.
The MRU list has empty rooms at beginning.

Reply:
Comment added.

Reply:
Isn't it possible to get it, then, via some sizeof() of a particular structure?

Reply:
I think nope.

Reply:
The MRU list of a shell bag has a non-continuous structure.

![image](https://github.com/reactos/reactos/assets/2107452/7cae5ca8-82d5-4532-ad36-ed4aeab4cb77)

Reply:
A shell bag is basically a data set for system management with a complex structure that is difficult for humans to analyze.